### PR TITLE
Added some meat and reminder

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): Week 4

## Description

Added some extra explanation and a reminder about the contributions.

### CODE REVIEW

File: docs/finalproject/week-4/lesson.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ title: Final Project - Week 4
 sidebar_label: Lesson
 ---
 
-See [Week 2](../week-2/lesson) for instructions for this week
+See [Week 2](../week-2/lesson) for instructions for this week. And make sure to check in again on the contribuitions per team members, as detailed on Week 3 [Week 3](../week-3/lesson)
```
Comment (SallyMcGrath) [inline docs/finalproject/week-4/lesson.md]:
```suggestion
See [Week 2](../week-2/lesson) for instructions for this week. 

Make sure to check in again on the contributions per team members, as detailed on Week 3 [Week 3](../week-3/lesson)
```

Please use spellcheck for submissions to the Syllabus. 🙏

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Minor changes

**Comment by kfklein15** [comment]
@SallyMcGrath , another one for your final approval.

File: docs/finalproject/week-4/lesson.md [hunk 2]
```diff
@@ -4,4 +4,6 @@ title: Final Project - Week 4
 sidebar_label: Lesson
 ---
 
-See [Week 2](../week-2/lesson) for instructions for this week
+See [Week 2](../week-2/lesson) for instructions for this week. 
+
+Make sure to check in again on the contributions per team members, as detailed on Week 3 [Week 3](../week-3/lesson)
```
Comment (SallyMcGrath) [inline docs/finalproject/week-4/lesson.md]:
```suggestion
Make sure to check in again on the contributions per team members, as detailed on [Week 3](../week-3/lesson)
```

Comment (SallyMcGrath) [reply docs/finalproject/week-4/lesson.md]: repeated phrase here

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Minor changes

**Review by SallyMcGrath — APPROVED** [review]
LGTM

**Pull request merged**